### PR TITLE
Fix elut correction factor

### PR DESCRIPTION
**Pull request by grazwegian**

### CODE REVIEW

File: stix/idl/processing/spectrogram/stx_convert_pixel_data.pro [hunk 1]
```diff
@@ -201,7 +201,7 @@ pro  stx_convert_pixel_data, fits_path_data = fits_path_data, fits_path_bk = fit
 
   spec_in = reform(spec_in,[dim_counts[0],n_detectors, n_times])
 
-  counts_spec =  spec_in[energy_bins,*, *]/ reform(reproduce(eff_ewidth, n_detectors*n_times),n_energies, n_detectors, n_times)
+  counts_spec =  spec_in[energy_bins,*, *] * reform(reproduce(eff_ewidth, n_detectors*n_times),n_energies, n_detectors, n_times)
```
Comment (samaloney) [inline stix/idl/processing/spectrogram/stx_convert_pixel_data.pro:204]: Oh no! Will this have any effect on the imaging?

**Review by samaloney — APPROVED** [review]
Looks good to me. I haven't checked but wondering is this "bug" relied on anywhere else specifically in the imaging?

File: stix/idl/processing/spectrogram/stx_convert_pixel_data.pro (hunk 1 above)
Comment (grazwegian) [reply stix/idl/processing/spectrogram/stx_convert_pixel_data.pro:204]: Fortunately for imaging the correction is done in a different way as it is performed on the summed imaging energy interval https://github.com/i4Ds/STIX-GSW/blob/42baf96dd783241c3bc0f4bcfad11e0f8ae01117/stix/idl/processing/pixel_data/stx_construct_pixel_data.pro#L221 .

**Comment by samaloney** [comment]
@grazwegian shall I merge this?

**Comment by grazwegian** [comment]
Yes, I think it's best to merge this now. @paolomassa has found a separate issue with the imaging ELUT correction but that will be dealt with in a different request.

**Pull request merged**